### PR TITLE
godoc reference card and minor alignment fix

### PR DESCRIPTION
**Pull request by pshail**
- Added godoc reference card
- Minor alignment/size fixes on other cards
Signed-off-by: Shailesh Pant <7968042+pshail@users.noreply.github.com>

**What this PR does / why we need it**:
Added godoc link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Kubernetes Cluster API Provider BYOH(Bring your own host)
-[![lint](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/lint.yml/badge.svg)](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/lint.yml)
-[![test](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/main.yml/badge.svg)](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/actions/workflows/main.yml)
+<p align="center">
```
Comment (shamsher31) [inline README.md:2]: I don't have a personal preference but most of the OSS projects keep it left aligned.
https://github.com/trending/go

**Review by shamsher31 — APPROVED** [review]
Looks good.
Screenshot 2021-11-11 at 3 08 22 PM
Screenshot 2021-11-11 at 3 08 13 PM

File: README.md (hunk 1 above)
Comment (pshail) [reply README.md:2]: @shamsher31 we can do that too, don't have any strong prefs there, was just trying to align with other CAPI-Providers

**Review by anusha94 — APPROVED** [review]
lgtm

**Pull request merged**